src/index.html

```diff
@@ -9,6 +9,62 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo-link">
+        <img
+          src="images/logo.png"
+          alt="logo"
```

Comment:
This text will be available for users
```suggestion
          alt="MOYO logo"
```